nvflare/app_common/workflows/initialize_global_weights.py

```diff
@@ -51,5 +50,5 @@ def __init__(
             min_responses_required=min_responses_required,
             wait_time_after_min_received=wait_time_after_min_received,
             timeout=task_timeout,
-            clients=clients,
+            clients=weights_client_name,
```

Comment:
@yanchengnv Can we support single strings here directly?

Reply:
Hence my question. I think we should support a single string in the underlying functions if the user wants to just target one client.

Reply:
Yes, if the `weight_method` is "first" but in the setting "client", the task should be only sent to that one client. The processor class (`GlobalWeightsInitializer `) expects the `client_name` to be just one string.

Reply:
Computing the clients argument now within the workflow. Should work for the different use cases described in the doc string.